speakers.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+title: Speaker List
+permalink: /speakers/
+---
+
+We are happy to annouce some of the speakers that will be at BSides Roanoke with more set to be added soon!
+
+## Rob Perry
+
+![Rob Perry Pic](images/robperry.jpeg)
+
+Robert Perry is the Chief Information Security Officer (CISO) for Carilion Clinic, one of the largest hospitals and healthcare providers based in southwest Virginia, serving nearly one million in the Virginia community and surrounding areas.
+
+As an expert in the technology industry, Mr. Perry’s charismatic speaking style and passion for technology and cybersecurity have made him a sought after speaker at conferences. Speaking topic range in areas including: cybersecurity systems, and tools, attack and counter attack cybersecurity response systems, authentication and access control.
+
+## Tom Weeks
+
+<img src="../images/tweeks.png" width="200" height="200">
+
+Tweeks (or Thomas Weeks) is the Director of Future Technology at Virginia Tech, Division of IT, and also sits as a Consulting Engineer at the Virginia Cyber Range where he creates cloud VM images for the Range, writes/edits Range labs, and works with Range authors to get their lab  content working in the Range.  Tweeks is a also a homeschooling technologist who's been teaching STEM / technology classes for 20+yrs, sits on the SW VA region's RBTC STEM and CyberSecurity committees and is very active in the NRV STEM education community. In his spare time he  likes playing with high power rockets as President of nrvr.org, electronics and really loves teaching kids how to peek behind the curtain of modern technology (LetsCodeBlacksburg.org).
+
+## Kennedy Toomey
+
+<img src="../images/kennedy.jpg" width="200" height="275">
+
+With a background in computer science, I transitioned to Application Security seamlessly after college. My goal is to make security easily accessible to everyone, through improving security culture and the developer experience. I believe that security should be a team that helps guide people forward instead of being a frustrating blocker. I am passionate about getting minorities, especially women, excited about security. 
```

Comment:
Adjusting voice to third person

```suggestion
With a background in computer science, Kennedy Toomey transitioned to Application Security seamlessly after college. Her goal is to make security easily accessible to everyone, through improving security culture and the developer experience. I believe that security should be a team that helps guide people forward instead of being a frustrating blocker. I am passionate about getting minorities, especially women, excited about security. 
```